npm/nameSpaceController.go

```diff
@@ -157,18 +149,6 @@ func (nsc *nameSpaceController) updateNamespace(old, new interface{}) {
 		}
 	}
 
-	nsKey := util.GetNSNameWithPrefix(key)
```

Comment:
Same keeping these filtering mechanisms in here would help churn through duplicates events faster with less code involved.